src/main/webapp/index.html

```diff
@@ -13,13 +13,5 @@
     <script src="scripts/popper.min.js"></script>
     <script src="scripts/bootstrap.bundle.min.js"></script>
```

Comment:
Nit: Remove these dependencies if they are not being used here. This page redirects immediately to  `/views/home.html`, right?